server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CachedBlobStore.java

```diff
@@ -210,6 +212,23 @@ public InputStream read(BucketName bucketName, BlobId blobId) {
         return backend.save(bucketName, inputStream, storagePolicy);
     }
 
+    @Override
+    public Publisher<BlobId> save(BucketName bucketName, ByteSource byteSource, StoragePolicy storagePolicy) {
+        Preconditions.checkNotNull(byteSource, "ByteSource must not be null");
+
+        if (isAbleToCache(bucketName, storagePolicy)) {
+            return Mono.from(backend.save(bucketName, byteSource, storagePolicy))
```

Comment:
why don't we call that `saveInCache` as for other `save` methods?

Reply:
Because the params are not exactly the same.

Reply:
I mean, you could create a method named `saveInCache` to make the code clearer _as we did for other methods_